modules/ROOT/pages/Development/BeginnersGuide/dependencies.adoc

```diff
@@ -19,54 +26,100 @@ If you get stuck in the video tutorials, return here and follow these guides ins
 Be sure to read over the steps here as well to be sure you're doing things correctly.
 ====
 
+== Before you Begin
+
+Make sure you've read the starting information on the
+xref:Development/BeginnersGuide/index.adoc[Getting Started title page]
+to learn important information about how to use these docs.
+
 == Satisfactory
 
-This one should be pretty straightforward, and odds are, you've already done it. Simply buy, download, and install Satisfactory from the Epic Games Launcher or Steam.
+This one should be pretty straightforward, and odds are, you've already done it.
+Simply buy, download, and install Satisfactory from the Epic Games Launcher or Steam.
 Launch the game at least once to ensure all files get set up correctly.
 
 == Visual Studio
 
-Visit the https://visualstudio.microsoft.com/downloads/[Microsoft Visual Studio downloads page] and download the version you want.
-We suggest either Visual Studio 2019 or 2017 Community, because they're free of charge. It is important that if you use 2019 you still install the https://visualstudio.microsoft.com/vs/older-downloads/[2017 compilers]. It may force you to sign up to access the older versions, but they are free.
+Visit the https://visualstudio.microsoft.com/downloads/[Microsoft Visual Studio downloads page]
+and download the version you want.
+We suggest either Visual Studio 2019 or 2017 Community,
+because they're free of charge.
+It is important that if you use 2019 you still install the https://visualstudio.microsoft.com/vs/older-downloads/[2017 compilers].
```

Comment:
Will wait to hear back before resolving this

Reply:
Just checked and I indeed don't have 2017 build tools installed. Works just fine
Can't do a suggestion because it only selects the latest line but can I change the snippet to recommend VS22 and remove the info about the build tools?

Reply:
Go for it, yeah